### DISCUSSION TITLE
iOS - Where is the toggle to enable experimental API to toggle tile cache in map view

### DISCUSSION THREAD

**Discussion by r3econ** [maplibre/maplibre-native#2626] [Q&A]
The newest iOS release https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.5.2 should include the experimental API to toggle tile cache in map view (https://github.com/maplibre/maplibre-native/pull/2590).

I updated the library, however, `MLNMapView` does not include the toggle to switch it on.
Do I need to somehow enable experimental features?

**Comment by louwers** [comment]
Let me have a look.

**Comment by r3econ** [comment]
@louwers thanks! Have you found something?
